cmd/neofs-node/config/node/config.go

```diff
@@ -0,0 +1,77 @@
+package nodeconfig
+
+import (
+	"errors"
+	"fmt"
+	"strconv"
+
+	"github.com/nspcc-dev/neofs-node/cmd/neofs-node/config"
+	"github.com/nspcc-dev/neofs-node/pkg/network"
+)
+
+const (
+	subsection = "node"
+
+	KeyDefault = ""
+
+	attributePrefix = "attribute"
+)
+
+var ErrEmptyAddress = errors.New("empty bootstrap address, see `node.address` section")
```

Comment:
Please don't create it for panic only.

Reply:
made that err private and made change test pkg to be the same pkg